### PR TITLE
Iris allparts

### PR DESCRIPTION
**Pull request by pp-mo**
Complete the separation of iris course sections, rationalise links and make all the 'suggested activities'  blocks look the same.

### CODE REVIEW

File: course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```
Comment (lbdreyer) [inline course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb]: Do we want to remove the execution of each cell?
I had added it was then it meant you could look through the notebook without needing to run it.

File: course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb [hunk 2]
```diff
@@ -375,16 +385,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`standard_name` is restricted to be a CF standard name (see the [CF standard name table](http://cfconventions.org/standard-names.html)). If there is not a suitable CF standard name, `cube.standard name` is set to `None` and the `long_name` is used instead. `long_name` is less restrictive and can be set to be any string. `var_name` is less commonly required and is used to help distinguish the phenomena.\n",
-    "\n",
+    "`standard_name` is restricted to be a CF standard name (see the [CF standard name table](http://cfconventions.org/standard-names.html)).  \n",
+    "If there is not a suitable CF standard name, `cube.standard name` is set to `None` and the `long_name` is used instead.  \n",
+    "`long_name` is less restrictive and can be set to be any string.  \n",
+    "`var_name` is less commonly required and is used to help distinguish the phenomena."
```
Comment (lbdreyer) [inline course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb]: Might be worth just updating this to say something like
`var_name` is used when loading in a netCDF file. It represents the name of the variable in the file.

File: course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb [hunk 3]
```diff
@@ -741,22 +752,35 @@
...
     "* Printing out a cube gives an overview of its metadata. We can get more information on the cube by inspecting its attributes (e.g. `cube.standard_name`, `cube.units()`)\n",
     "* A coordinate has a similar interface to a cube, but a coordinate has points and bounds, where a cube has data."
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (lbdreyer) [inline course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb]: We can get rid of this extra code cell.

**Comment by lbdreyer** [comment]
@pp-mo I have made a couple of minor comments, but I am tempted to merge this in as we then have something to both work off.

File: course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb (hunk 1 above)
Comment (pp-mo) [reply course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb]: Just noticed, I never replied to this.
I actually think it is better to have the users 'discover' all the outputs.
It also makes all the management operations on GitHub far easier (i.e. seeing changes + reviewing).

**Comment by pp-mo** [comment]
> @lbdreyer I have made a couple of minor comments, but I am tempted to merge this

Thanks, see #156

**Pull request merged**